docs/markdown/Contributions/development/style-guide.md

```diff
@@ -406,3 +406,35 @@ class TestDemo(unittest.TestCase):
     def test_demo(self) -> None:
         self.assertEqual(y, 2)
 ```
+
+Documentation
+-------------
+
+User documentation uploaded to the [Pantsbuild web docs site](https://www.pantsbuild.org/docs) consists of two sections:
+* the reference docs that are generated from docstrings in the source code
+* the guides that are generated from the `docs/` directory's Markdown files.
+
+### Reference docs
+
+Not every docstring will make it to the website: currently only docstrings for global options, goals, subsystems, and targets are published. Please be extra vigilant when writing these and remember that docstrings are going to be rendered as Markdown. 
+
+It may be helpful to consider the following:
+
+* use the `softwrap` helper function to turn a multiline string into a softwrapped string
+* if you experience `mypy` typing issues using the `softwrap` for documenting subclasses of `Field` and `Target` classes, consider using the `help_text` convenience function
+* text inside the angle brackets (e.g. `<value>`) will be ignored when rendered if not wrapped in backticks
+* to create a numbered or bullet list, use 2 space indentation
```

Comment:
or use
https://github.com/pantsbuild/pants/blob/ab99aec1a2366f37438416e109c46973a9970f14/src/python/pants/util/strutil.py#L186-L194

Reply:
oh brilliant, updated the instructions.